rel-1.0.0/debian/core/Dockerfile

```diff
@@ -1,9 +1,7 @@
 FROM microsoft/dotnet-nightly:rel-1.0.0-core-deps
 
 RUN apt-get update \
-    && apt-get install -y --no-install-recommends \
-        ca-certificates \
-        curl \
```

Comment:
>curl \ [](start = 8, length = 6)

My preference would be to leave `curl \` on a separate line.  It makes it visually easier to read the packages that are installed and makes adding/removing packages easier to diff.